src/3rd_party/CMakeLists.txt

```diff
@@ -66,10 +66,6 @@ if(USE_SENTENCEPIECE)
   set(SPM_ENABLE_TCMALLOC ON CACHE BOOL "Enable TCMalloc if available.")
 
   if(USE_STATIC_LIBS)
-    message(WARNING "You are compiling SentencePiece binaries with -DUSE_STATIC_LIBS=on. \
```

Comment:
Have you checked if it works now? Asking because the PR description does not mention this change.

Reply:
Yes. works fine.

Reply:
😌💖

Reply:
Added to "fixed" Changelog section